test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -283,6 +283,25 @@ def _init_test_roi_heads_faster_rcnn():
         box_score_thresh, box_nms_thresh, box_detections_per_img)
     return roi_heads
 
+def _construct_tensor_for_quantization_test(shape, offset=None, max_val=None):
+    """Due to difference in implementation details between PyTorch and ONNXRuntime, randomly generated
+    test data for quantization tests can be flaky. To help stablize the test, this helper function is
+    used to generate weights and test inputs in a deterministic way.
+
+    Args:
+        shape (Tuple[int]): Shape for tensor to construct.
```

Comment:
Adding the type annotations inline will help IDEs analyze the code. Type checkers like pylance will use this info to display errors inline if there is a typing issue. This will ultimately help us understand the code better and code for efficiently. Happy to discuss further.